tcl/linuxcnc.tcl.in

```diff
@@ -76,7 +76,7 @@ proc hal {args} {
 # called *.msg, (e.g. en_US.msg).
 package require msgcat
 if {$linuxcnc::_langinit && [info exists env(LANG)]} {
```

Comment:
It might be that this block should **not** be conditional on the LANG variable setting any longer.  For instance, in other programs it seems to be enough to set LC_MESSAGES:
```
$ (unset LANG; LC_MESSAGES=fr_FR.UTF-8 rm /)
rm: impossible de supprimer '/': est un dossier
```
However, I am not familiar with how Tcl works in this case so it could behave differently.